src/mustache.erl

```diff
@@ -285,14 +285,24 @@ escape_char($') -> <<"&apos;">>;
 escape_char(C)  -> <<C:8>>.
 
 %% @doc fetch the value of the specified key from {@link data/0}
--spec data_get(data_key(), data(), Default :: term()) -> term(). 
+-spec data_get_(data_key(), data(), Default :: term()) -> term().
```

Comment:
Maybe this will fail to compile.

Please modified this.
`data_get_` -> `data_get`